### PR TITLE
Custom cutout thumbnail transformation logic for PGIR

### PR DESCRIPTION
**Pull request by dekishalay**
This PR modifies the thumbnail generation logic for the specific ZTF and PGIR streams (so that North is up and West is on the right). 
Separated the thumbnail manipulation logics for ZTF and PGIR.

### CODE REVIEW

File: kowalski/alert_broker.py [hunk 1]
```diff
@@ -447,10 +447,18 @@ def make_thumbnail(
...
+                    # header = hdu[0].header
+                    data_flipped_y = np.flipud(hdu[0].data)
+                    data_corrected = data_flipped_y
+        elif self.instrument == "PGIR":
+            cutout_data = cutout_data["stampData"]
+            with gzip.open(io.BytesIO(cutout_data), "rb") as f:
+                with fits.open(io.BytesIO(f.read()), ignore_missing_simple=True) as hdu:
+                    # header = hdu[0].header
+                    data_flipped_x = np.fliplr(hdu[0].data)
+                    data_corrected = np.rot90(data_flipped_x, 3)
```
Comment (dmitryduev) [inline kowalski/alert_broker.py:461]: A lot of unnecessary duplication.
```suggestion
        with gzip.open(io.BytesIO(cutout_data), "rb") as f:
            with fits.open(io.BytesIO(f.read()), ignore_missing_simple=True) as hdu:
                image_data = hdu[0].data

        if self.instrument == "ZTF":
            image_data = np.flipud(image_data)
        elif self.instrument == "PGIR":
            image_data = np.rot90(np.fliplr(image_data), 3)
```

File: kowalski/alert_broker.py [hunk 2]
```diff
@@ -461,7 +469,7 @@ def make_thumbnail(
         fig.add_axes(ax)
 
         # replace nans with median:
-        img = np.array(data_flipped_y)
+        img = np.array(data_corrected)
```
Comment (dmitryduev) [inline kowalski/alert_broker.py:472]:
```suggestion
        img = np.array(image_data)
```

**Review by dmitryduev — CHANGES_REQUESTED** [review]
Thanks @dekishalay - please see my suggestions and do check the CI (e.g. on your last commit - https://github.com/dmitryduev/kowalski/pull/142/checks?check_run_id=3509707299).

**Comment by dmitryduev** [comment]
`TNS monitoring` test is expected to fail on a non-master branch, ignore it on the CI.

File: kowalski/alert_broker.py (hunk 2 above)
Comment (dekishalay) [reply kowalski/alert_broker.py:472]: This looks fine.

File: kowalski/alert_broker.py (hunk 1 above)
Comment (dekishalay) [reply kowalski/alert_broker.py:461]: Looks good!

**Review by dmitryduev — APPROVED** [review]
LGTM! Failure on the CI is expected (need to handle that properly, BTW, but that's on me I guess..), so merging.

**Pull request merged**